uptime_service_validation/coordinator/server.py

```diff
@@ -235,6 +235,10 @@ def setUpValidatorPods(time_intervals, logging, worker_image, worker_tag):
             volume_mounts=[auth_volume_mount],
         )
 
+        nodepool = os.environ.get("K8S_NODE_POOL")
```

Comment:
**Praise:** Great for parametrizing this.

Reply:
Good idea! Done. 👍